tests/framework/utils/k8s_helper.go

```diff
@@ -227,6 +227,9 @@ func (k8sh *K8sHelper) ExecToolboxWithRetry(retries int, namespace, command stri
 // ResourceOperation performs a kubectl action on a pod definition
 func (k8sh *K8sHelper) ResourceOperation(action string, manifest string) error {
 	args := []string{action, "-f", "-"}
+	if action == "apply" {
+		args = []string{"apply", "--server-side=true", "-force-conflicts", "-f", "-"}
```

Comment:
Is this extra `-` I'm suggesting necessary? Or does kubectl accept single `-` flags?

```suggestion
		args = []string{"apply", "--server-side=true", "--force-conflicts", "-f", "-"}
```

Reply:
THanks @BlaineEXE https://github.com/rook/rook/actions/runs/4244696812/jobs/7379183150

THe upgrade test is in good shape now:)